start-e2e.sh

```diff
@@ -5,21 +5,25 @@
 # SPDX-License-Identifier: EUPL-1.2+
 #
 
-# Change Directory to your test directory
+set -e
+
+echo "Change Directory to e2e test directory .."
 cd ./src/e2e
-# Install dependencies
+echo "Install dependencies ..."
 npm ci
 
-# Install Playwright Browsers
+echo "Install Playwright Browsers ..."
 npx playwright install --with-deps
 
-# Define world-parameters JSON
-world_params='{"urls": { "zac": "'$ZAC_URL'", "openForms": "'$OPEN_FORMS_URL'"}, "headless": false, "users": {"Bob": {"username": "'$E2E_TEST_USER_1_USERNAME'", "password": "'$E2E_TEST_USER_1_PASSWORD'"}, "Oscar": {"username": "'$E2E_TEST_USER_2_USERNAME'", "password": "'$E2E_TEST_USER_2_PASSWORD'"}}}'
+echo "Cleanup screenshots ..."
+rm -rf reports/screenshots/* reports/videos/*
```

Comment:
Different thing maybe but shouldn't we also extend our `gradlew clean` task to clean up this e2e stuff?

```delete("$rootDir/src/e2e/reports")```

Reply:
I think I would also be in favor to use `/build` for all build/generated frontend code instead of the current folders but that's for another day I guess. Now we have frontend build and reports stuff deep down in our tree which makes it hard to even notice them.

Reply:
Added to gradle build